content/docs/command-reference/plots/show.md

```diff
@@ -349,10 +349,10 @@ file:///Users/usr/src/dvc_plots/index.html
 In many automation scenarios (like
 [CI/CD for ML](/doc/use-cases/ci-cd-for-machine-learning)), it is convenient to
 output the [Vega-Lite](https://vega.github.io/vega-lite/) spec file instead of
-rendering an HTML plot. For example, to generating another image format like PNG
-or JPEG, or to include it differently into a web/mobile app. The `--show-vega`
-option prevents wrapping this plot spec in HTML, printing the resulting JSON to
-standard output instead:
+rendering an HTML plot. For example, to generate another image format (like PNG,
+JPEG, or SVG) or to include it differently into a web/mobile app. The
```

Comment:
Thanks for fixing the typo

I wouldn't add SVG to this list though since it's not meant to be exhaustive, and just 2 examples seem better than 3 out of 4.

```suggestion
rendering an HTML plot. For example, to generate another image format (like PNG
or JPEG) or to include it differently into a web/mobile app. The
```